tests/command.help.test.js

```diff
@@ -130,3 +130,20 @@ test('when both help flags masked then not displayed in helpInformation', () =>
   const helpInformation = program.helpInformation();
   expect(helpInformation).not.toMatch('display help');
 });
+
+test('when default value is object with toString and displays in helpInformation', () => {
+  const program = new commander.Command();
+  program
+    .option('--host <host>', 'select host', new URL('http://example.com/'));
+  const helpInformation = program.helpInformation();
+  expect(helpInformation).toContain('http://example.com/');
+});
+
+test('when default value is object and displays in helpInformation', () => {
```

Comment:
e.g. 'when default value is object without custom toString then json displays in helpInformation'